src/RA_Integration.vcxproj

```diff
@@ -329,7 +329,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Analysis|Win32'">
     <TargetName>RA_Integration</TargetName>
-    <CodeAnalysisRuleSet>CppCoreCheckClassRules.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>CppCoreCheckConstRules.ruleset</CodeAnalysisRuleSet>
```

Comment:
Is there some way to enforce both? i.e. `CppCoreCheckClassRules.ruleset,CppCoreCheckConstRules.ruleset`?

Reply:
You can, but it'd have to be a custom ruleset. The "CppCoreCheckRules" one shows too much (over 1000). I can try doing that.